client/lib/react-helpers/index.js

```diff
@@ -9,15 +9,25 @@ import { Provider as ReduxProvider } from 'react-redux';
  * Internal dependencies
  */
 import analytics from 'lib/analytics';
+import Stylizer, { insertCss } from './stylizer';
 
 export function concatTitle( ...parts ) {
 	return parts.join( ' › ' );
 }
 
-export function renderPage( context, component ) {
-	renderWithReduxStore(
-		component,
-		document.getElementById( 'primary' ),
+export function renderPage( component, domContainer, context ) {
```

Comment:
Yes it would!
My goal here was to change the least amount of code possible, doing search and replace only.
Making the optional parameter the last would indeed be a cleaner solution, I'll find the time to update the PR but it's going to be a lot of manual changes :)